relay-general/src/store/mod.rs

```diff
@@ -31,6 +32,8 @@ pub struct StoreConfig {
     pub protocol_version: Option<String>,
     pub grouping_config: Option<Value>,
     pub user_agent: Option<String>,
+    pub received_at: Option<DateTime<Utc>>,
```

Comment:
This is only optional to retain `Default for StoreConfig`. Effectively, it is always set in the event processor. Internally, `StoreProcessor` defaults to `Utc::now` if we ever choose to not pass a value here.